### PR TITLE
docs: improve readme

### PR DESCRIPTION
**Pull request by VincentRPS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+become fully open source, but for now, it's far too undeveloped to require an open source license.
 
 ## Development
 
 Use nightly Rust.
 
-If you want to add a dependency, add it to the workspace `Cargo.toml` and then
-in the crate that you want to use this dependency in specify that dependency as
-`dep_name = { workspace = true }`. This makes updating deps much easier.
+If you want to add a dependency, add it to the workspace Cargo.toml and then in the crate that you want to use this dependency in specify that dependency as dep_name = { workspace = true }. This makes updating deps much easier.
```
Comment (elenakrittik) [inline README.md:14]: Why did you remove backticks?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,14 @@
 # Metal
 
-A programming language. We'll add a catchy slogan soon, pinky swear
+Prototype compiled language built in Rust with LLVM. We'll add a catchy slogan later.
```
Comment (elenakrittik) [inline README.md:3]: If you feel like it, feel free to remove the latter sentence, no need to make compromises here :D

File: README.md [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
+
+## Contributing
+
+Currently, Metal is more or less under source availability. Metal will eventually
+become fully open source, but for now, it's far too undeveloped to require an open source license.
```
Comment (elenakrittik) [inline README.md:8]: The "too underdeveloped to require an OSS licence" part sounds strange to me. No matter the degree of "developedness" of a piece of software, there's no point at which it requires to be OSS, right? Also, i think "source availability" suggests more freedom than absence of a license means (if not for GitHub's ToS, no one could even legally *view* the code). I propose leaving out this section until we decide on a license

**Review by elenakrittik — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (VincentRPS) [reply README.md:14]: Byproduct of accidentally removing it then adding it back

File: README.md (hunk 3 above)
Comment (VincentRPS) [inline README.md:8]:
```suggestion
become fully open source, but for now, since we're the only ones
y
```

File: README.md [hunk 4]
```diff
@@ -1,11 +1,14 @@
 # Metal
 
-A programming language. We'll add a catchy slogan soon, pinky swear
+Prototype compiled language built in Rust with LLVM. We'll add a catchy slogan later.
+
+## Contributing
+
+Currently, Metal is more or less under source availability. Metal will eventually
```
Comment (VincentRPS) [inline README.md:7]:
```suggestion
Currently, Metal is unlicensed and solely owned by Vincent and Lena the maintainers. Metal will eventually open up and
```

File: README.md [hunk 5]
```diff
@@ -1,11 +1,14 @@
 # Metal
 
-A programming language. We'll add a catchy slogan soon, pinky swear
+Prototype compiled language built in Rust with LLVM. We'll add a catchy slogan later.
+
+## Contributing
+
+Currently, Metal is unlicensed and solely owned by Vincent and Lena the maintainers. Metal will eventually open up and
```
Comment (elenakrittik) [inline README.md:7]: > owned by <us>
https://choosealicense.com/no-permission/

So say that i believe we'll at the very least need a LICENSE file that says "Copyright <us>. All rights reserved"

**Review by elenakrittik — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -1,11 +1,16 @@
 # Metal
 
-A programming language. We'll add a catchy slogan soon, pinky swear
+Prototype compiled language built in Rust with LLVM.
+
+## Contributing
+
+Currently, Metal is currently not open source software. Metal will eventually open up and
+become fully open source, but for now, since we're the only ones using this language it's fully closed.
```
Comment (elenakrittik) [inline README.md:8]:
```suggestion
become fully open source, but for the time being, it's not open for contributions.
```

File: README.md [hunk 7]
```diff
@@ -1,11 +1,16 @@
...
 
 ## Development
 
 Use nightly Rust.
 
-If you want to add a dependency, add it to the workspace `Cargo.toml` and then
-in the crate that you want to use this dependency in specify that dependency as
-`dep_name = { workspace = true }`. This makes updating deps much easier.
+If you want to add a dependency, add it to the workspace Cargo.toml and then
+in the crate that you want to use this dependency in specify that dependency as dep_name = { workspace = true }.
```
Comment (elenakrittik) [inline README.md:15]:
```suggestion
in the crate that you want to use this dependency in specify that dependency as `dep_name = { workspace = true }`.
```

**Review by elenakrittik — CHANGES_REQUESTED** [review]

**Review by elenakrittik — APPROVED** [review]
